### PR TITLE
chore(server): fix mongo connection to allow cloud connect

### PR DESCRIPTION
**Pull request by Mackbellemore**

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-kgyp4q Deployment Pipeline](https://app.clickup.com/t/kgyp4q)

**Comment by Mackbellemore** [comment]
> empty quote?

nay its eslint complaining about a require vs import but i think i just need to disable it

File: server/config/default.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
+// eslint-disable-next-line
```
Comment (gordonpn) [inline server/config/default.js:1]: LOL does it npm run build and npm run start? If that works then fine by me

Comment (Mackbellemore) [reply server/config/default.js:1]: we need this because the .env file cant be placed in server folder or else it wont resolve the path

Comment (Mackbellemore) [reply server/config/default.js:1]: and ya it builds and runs

Comment (Mackbellemore) [reply server/config/default.js:1]: this may break the deployment actually fuck, depends on where and how the env variables are resolved to

Comment (gordonpn) [reply server/config/default.js:1]: doesn’t process.env fallback to the shell environment if an env file isn’t available?

Comment (Mackbellemore) [reply server/config/default.js:1]: true, so deployment should be ok it was working earlier. This fixes local use

**Review by gordonpn — APPROVED** [review]

**Pull request merged**